react/src/components/StorageStatusPanelCard.tsx

```diff
@@ -92,96 +98,106 @@ const StorageStatusPanelCard: React.FC<StorageStatusPanelProps> = ({
+          >
+            <BAIPanelItem
+              title={
+                count > 0 ? (
+                  // Add a tag to the Tooltip to make it clickable
+                  // eslint-disable-next-line
+                  <a
+                    onClick={() => {
+                      toggleInvitationResponseModal();
+                    }}
                   >
-                    <Badge
-                      count={count > 0 ? `+${count}` : null}
-                      offset={[0, -`${token.sizeXS}`]}
+                    <Tooltip
+                      title={
+                        count > 0
+                          ? t('data.InvitedFoldersTooltip', {
+                              count: count,
+                            })
+                          : null
+                      }
+                      rootClassName={styles.invitationTooltip}
+                      placement="topRight"
                     >
-                      <Typography.Title level={5} style={{ margin: 0 }}>
-                        {t('data.InvitedFolders')}
-                      </Typography.Title>
-                    </Badge>
-                  </Tooltip>
-                </a>
-              ) : (
-                <Typography.Title level={5} style={{ margin: 0 }}>
-                  {t('data.InvitedFolders')}
-                </Typography.Title>
-              )
-            }
-            value={
-              <Typography.Text
-                strong
-                style={{
-                  fontSize: token.fontSizeHeading1,
-                  color: token.Layout?.headerBg,
-                }}
-              >
-                {invitedCount}
-              </Typography.Text>
-            }
-          />
-        </Col>
-      </Row>
-    </BAICard>
+                      <Badge
+                        count={count > 0 ? `+${count}` : null}
+                        offset={[0, -`${token.sizeXS}`]}
+                      >
+                        <Typography.Title level={5} style={{ margin: 0 }}>
+                          {t('data.InvitedFolders')}
+                        </Typography.Title>
+                      </Badge>
+                    </Tooltip>
+                  </a>
+                ) : (
+                  <Typography.Title level={5} style={{ margin: 0 }}>
+                    {t('data.InvitedFolders')}
+                  </Typography.Title>
+                )
+              }
+              value={
+                <Typography.Text
+                  strong
+                  style={{
+                    fontSize: token.fontSizeHeading1,
+                    color: token.Layout?.headerBg,
+                  }}
+                >
+                  {invitedCount}
+                </Typography.Text>
+              }
+            />
+          </Col>
+        </Row>
+      </BAICard>
+      <FolderInvitationResponseModal
+        open={isInvitationResponseModalOpen}
+        onRequestClose={toggleInvitationResponseModal}
+      />
```

Comment:
Depending on the success of the invitation, a refresh of the vfolder node list will likely be required.